### DISCUSSION TITLE
Cart feature question

### DISCUSSION THREAD

**Discussion by hamzaelmaghari** [bavix/laravel-wallet#807] [Q&A] [answered]
Hello devs!

I have question about the cart feature, is it dynamic for each user? I mean does it relying on session data? I want to implement a Cart model that rely on database to keep that data saved ... if I went trough this situation, should I extend `laravel-wallet` Cart class? or just keep my file stand alone?

Thanks 🙏

**Comment by rez1dent3** [comment]
Hello. There are no sessions in the cart, objects must be stored somewhere on your side and then added to the cart (each time).

**Reply by hamzaelmaghari** [reply]
Hey friend!
Thanks for your answer, do you mind providing me an example if you could?

**Reply by rez1dent3** [reply, marked as answer]
You store the cart somewhere and then fill it with the necessary items and quantities before purchasing.

If I understand your task correctly, then the example from the documentation suits you:
https://bavix.github.io/laravel-wallet/#/cart?id=fill-the-cart
